### PR TITLE
Digital Portfolio Creation

### PR DESCRIPTION
**Pull request by ogwurujohnson**
- make sure less works

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,19 +1,139 @@
...
+                  <em class="children">"go"</em>
+                <em class="closing-braces">]</em>,
+                <em class="parent">"dependencies"</em>: {
+                  <em class="parent-children">"react"</em><em class="colon">:</em> <em class="children">"^16.6.3"</em>,
+                  <em class="parent-children">"webpack"</em>: <em class="children">"4.19.1"</em>,
+                  <em class="parent-children">"express"</em>: <em class="children">^4.16.4</em>
+                <em class="closing-braces">}</em>,
+                <em class="parent">"devDependencies"</em>: {
+                  <em class="parent-children">"tea"</em>: <em class="children">"^1.0.0"</em>,
+                  <em class="parent-children">"music"</em>: <em class="children">t-series :D</em>,
```
Comment (mattshardman) [inline index.html]: The t-series is not valid json lol .... joking. Love this bit!

**Comment by mattshardman** [comment]
Johnson this is great! Love it!

* You have completed all the mvp
* Your code is really neat and nicely organised (good job splitting it into separate files)
* Your git work flow is superb as always
* Love the package.json section lol!
* Great job including particle.js as well
* You've added google fonts

Overall awesome job, you clearly have a good grasp of less and have gone well above the MVP. You should volunteer to show stuff like this is standup its awesome!

File: index.html (hunk 1 above)
Comment (ogwurujohnson) [reply index.html]: lol, thanks

**Comment by ogwurujohnson** [comment]
> Johnson this is great! Love it!
> 
> * You have completed all the mvp
> * Your code is really neat and nicely organised (good job splitting it into separate files)
> * Your git work flow is superb as always
> * Love the package.json section lol!
> * Great job including particle.js as well
> * You've added google fonts
> 
> Overall awesome job, you clearly have a good grasp of less and have gone well above the MVP. You should volunteer to show stuff like this is standup its awesome!

Thanks, Matt for the kind words, really appreciate the encouragement

**Pull request merged**